.travis.yml

```diff
@@ -36,6 +36,7 @@ install:
     else
         pip install -U git+https://github.com/asmeurer/jupyter_console@display_completions;
         pip install -U "pexpect>=3.3" pyflakes pytest rlipython requests jupyter flaky flake8 'notebook<6.1';
+        pip install 'prompt_toolkit<3.0.6';
```

Comment:
Should this be on the previous line? Also, what happened in 3.0.6?

Reply:
It will be easier to remove if it's on it's own line :-) 

I'm not sure what happened in 3.0.6, I had a quick look and it seem like there are better detection of whether it's running in a tty, and I think it's trying to not emit colors or things like that; so the pyflyby pexpect might be waiting forever for a control sequence which does not appear.

Reply:
Yeah there's a function in the interactive tests that removes all the ANSI escapes that has to be updated every time prompt toolkit is updated. There really needs to be a better way to do it.

Reply:
See #32